CMakeLists.txt

```
@@ -23,7 +23,7 @@ endif()
 
 project(torch-mlir LANGUAGES CXX C)
 set(CMAKE_C_STANDARD 11)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ standard to use")
```

How about just using 20 here if this is what we want? Then also CI builds of this repo will use the version we want?
See also my comment on onnx-mlir
```suggestion
set(CMAKE_CXX_STANDARD 20)
```